servlet-dist-legacy/pom.xml

```diff
@@ -175,6 +175,91 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>1.7</version>
```

Comment:
The version should be a property, if it's not already, in the main pom.

Reply:
Fixed.